### PR TITLE
pulled from main

### PR DESCRIPTION
**Pull request by Harinze**

### CODE REVIEW

File: apps/Client/src/pages/Auth/Login/LoginPage.tsx [hunk 1]
```diff
@@ -1,3 +1,158 @@
...
+//           </div>
+//         </form>
+//       </div>
+//       <ToastContainer position="top-right" autoClose={3000} hideProgressBar />
+//     </div>
+//   );
+// };
+
+// export default LoginPage;
+
```
Comment (Don-christo) [inline apps/Client/src/pages/Auth/Login/LoginPage.tsx:154]: Why all the comments? If you're not using it, why not remove it entirely?

**Review by Don-christo — APPROVED** [review]
Good job. Thank you. Please remove the comments. There's no use for them, and why are you changing the folder names again? These folder names have been one of the things causing issues. Why changing them again?

**Review by Don-christo — CHANGES_REQUESTED** [review]
Why are you changing the folder names?

**Comment by Harinze** [comment]
I was errors with the folder names

**Review by Don-christo — APPROVED** [review]
Good job, boss.

**Pull request merged**